en/api/perl.rst

```diff
@@ -24,9 +24,9 @@ Installing and Configuring Perl
 
 *   Perl: It is recommended to use an appropriate version of Perl based on your system environment. For example, all Linux and FreeBSD distributions come with Perl. For Windows, ActivePerl is recommended. For details, see https://www.activestate.com/products/perl/ .
 
-*   CUBRID: To build CUBRID Perl driver, you need to get the CCI driver. You can get it from installing CUBRID. You can download the CUBRID Perl driver's source code from https://www.cubrid.org/downloads .
+*   CUBRID: To build CUBRID Perl driver, you need to get the CCI driver. You can get it from installing CUBRID Engine. It can be downloaded from https://www.cubrid.org/downloads. After downloading/installing proper version of CUBRID Engine, you can find required CCI driver from the installed CUBRID Engine, $CUBRID/cci.
```

Comment:
```suggestion
*   CUBRID: To build the CUBRID Perl driver, the CCI driver is required. You can obtain it by installing the CUBRID Engine, which can be downloaded from https://www.cubrid.org/downloads. After downloading and installing a proper version of the CUBRID Engine, you can find the required CCI driver in the installed CUBRID Engine directory, $CUBRID/cci.
```

Reply:
Thank you for your comment!
I'll accept your suggestion.